rnaseq-star-align/star2pass.rnaseq_harmonization.cwl

```diff
@@ -0,0 +1,114 @@
+cwlVersion: v1.0
+class: Workflow
+id: gdc_rnaseq_star2pass_wf
+requirements:
+  - class: SubworkflowFeatureRequirement
+  - class: SchemaDefRequirement
+    types:
+      - $import: /tools/readgroup.cwl
```

Comment:
does it need `./tools` ?

Reply:
IMO a root-level `tools` directory is needed because nearly every workflow in the "tree" references a tooling workflow. I want to avoid needing to specify `../../../tools` because those references would break if the workflow is moved.